public/viewjs/consume.js

```diff
@@ -1,4 +1,4 @@
-﻿$('#save-consume-button').on('click', function(e)
+$('#save-consume-button').on('click', function(e)
```

Comment:
This is just removing the UTF-8 BOM since my editor kept doing that.

Reply:
I haven't blamed that, I've blamed changing single quotes to double quotes for no reason (not in that line this particular comment was pinned on).